### ISSUE TITLE
Title text for tags is pulling from posts

### ISSUE THREAD

**Issue by stevejburge** [TaxoPress/Tag-Groups#110]
The Alt text should be more on the lines of:

"Posts labeled with the ABC tag"

![Screenshot 2024-04-03 at 11 52 42 AM](https://github.com/TaxoPress/Tag-Groups/assets/3868028/4e8556cd-b44e-43c7-9e23-baa07b094e83)
